clientv3/balancer.go

```diff
@@ -66,18 +66,21 @@ func newSimpleBalancer(eps []string) *simpleBalancer {
 	for i := range eps {
 		addrs[i].Addr = getHost(eps[i])
 	}
-	notifyCh <- addrs
 	sb := &simpleBalancer{
 		addrs:    addrs,
 		notifyCh: notifyCh,
 		readyc:   make(chan struct{}),
-		upEps:    make(map[string]struct{}),
 		upc:      make(chan struct{}),
 		host2ep:  getHost2ep(eps),
 	}
+	sb.notifyAddrs(addrs)
 	return sb
 }
 
+func (b *simpleBalancer) notifyAddrs(addrs []grpc.Address) {
```

Comment:
this function seems superfluous; what's wrong with inlining `b.notifyCh <- addrs`?